client/src/components/shared/SideNav/SideNav.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from "react";
+import { Link, useLocation } from "react-router-dom";
+import { Menu } from "antd";
+
+import { Web3Context } from "../../../contexts/RootContexts";
+import Web3SignIn from "../../account/Web3SignIn";
+
+import Icons from "../../../Icons";
+
+import "./SideNav.scss";
+
+const SideNav = () => {
+  const [web3] = useContext(Web3Context);
+  const location = useLocation();
+
+  return (
+    <>
+      <Menu
+        defaultSelectedKeys={location.pathname}
+        mode="inline"
+        inlineIndent={0}
+      >
+        <div className="web3feedback">
+          {web3 && web3.account ? null : <Web3SignIn size="small" />}
```

Comment:
Can we display the account address the user is currently logged in with on the page somewhere?